CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [unreleased]
+### Changed
```

Comment:
Do you guys think we should update below logic as well? prbabbly change it from 10 to 2. 
https://github.com/perpetual-protocol/perp-curie-contract/blob/main/contracts/lib/UniswapV3Broker.sol#L240

Reply:
Yes, was thinking about it. Changing the `10` might be fine, though I'm not sure whether there is some limitation on Uniswap pool side?

Though the main reason we didn't do that in this PR is because changing `UniswapV3Broker` will upgrade basically all our contracts, and we would like to make this release simpler since it blocks Kantaban.

Reply:
Will changing 10 to 2 help anything? Would like to narrow the scope of this release and make things easier

Reply:
Unless we want to propose to change market twap from 15s to less than 10 then yes we have to change this

Reply:
@vinta got it, if the change needs to upgrade many things, then it's fine.